### PR TITLE
[WIP] Update baremetalds e2e test to use a proxy

### PR DESCRIPTION
**Pull request by kirankt**
Currently the baremetalds-e2e-test workflow copies the openshift-tests binary and then utilizes ssh to connect to the remote host and run the binary. Since the addition of a proxy to the IPI tests, we should attempt to use the proxy to connect to the remote host. This PR attempts to use the proxy and removes remote execution of binaries using ssh.

### CODE REVIEW

**Comment by stbenjam** [comment]
It's too big to test :-( I'd maybe just remove all the test skips, that should make it small enough to run. We'll get a good signal about the proxy approach, even if the now unskipped tests fail.

File: ci-operator/step-registry/baremetalds/e2ev2/baremetalds-e2ev2-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: baremetalds-e2ev2
+  steps:
+    pre:
+      - chain: baremetalds-ipi-pre
+    test:
+      - ref: baremetalds-e2e-conf
+      - ref: baremetalds-e2ev2-test
```
Comment (andfasano) [inline ci-operator/step-registry/baremetalds/e2ev2/baremetalds-e2ev2-workflow.yaml:8]: nit: it's sufficient the e2ev2 step, the workflow could be defined directly at the job level

**Comment by kirankt** [comment]
Replace by: https://github.com/openshift/release/pull/19987

**Pull request closed without merging**